doc/signal_path.rst

```diff
@@ -0,0 +1,302 @@
+efficiency than a purely frequency-domain approach (which would require the
+PFB to be run on the full bandwidth), while mitigating many of the filter
+design problems inherent in a purely time-domain approach (the roll-off of the
+FIR filter can be hidden in the discarded outer channels).
+
+The figure below shows the modified signal path.
+
+.. tikz:: Signal path for narrow-band channelisation (with new stages in blue).
+   :libs: chains, positioning
+
+   \tikzset{
+     base/.style={minimum width=2.5cm, minimum height=1cm, align=center},
+     op/.style={draw, base},
+     extra/.style={draw=blue, color=blue},
+     control/.style={draw, base, rounded corners, dotted},
+     lbl/.style={font=\scriptsize},
+     every join/.style={draw,->},
+     >=latex,
+   }
+   \newcommand{\side}[2]{
+     \node[op, extra, on chain, join=by {#2, edge label=cf32}] (ddc) {DDC};
+     \node[op, on chain, join=by {#2, edge label=cf32}] (cdelay#1) {Coarse delay};
+     \node[op, on chain, join=by {#2, edge label=cf32}] (pfb#1) {PFB};
+     \node[op, extra, on chain, join=by {#2, edge label=cf32}] (discard#1) {Discard\\ channels};
+     \node[op, on chain, join=by {#2, edge label=cf32}] (eq#1) {Fine delay\\ Equalisation};
+     \node[op, on chain, join=by {#2, edge label=cf32}] (dither#1) {Dither};
+     \node[op, on chain, join=by {#2, edge label=cf32}] (quant#1) {Quantise};
+   }
+   \node[op] (receive) {Receive};
+   \begin{scope}[start chain=chainx going below]
+     \node[op, below left=of receive, on chain] (unpackx) {Unpack};
+     \side{x}{lbl, swap}
+   \end{scope}
+   \begin{scope}[start chain=chainy going below]
+     \node[op, below right=of receive, on chain] (unpacky) {Unpack};
+     \side{y}{lbl}
+   \end{scope}
+   \begin{scope}[start chain=sink going below]
+     \node[op, on chain, below right=of quantx] (pack) {Corner turn\\ Pack};
+     \node[op, on chain, join=by {lbl, edge label=ci8}] (transmit) {Transmit};
+   \end{scope}
+   \node[control, right=of pfbx] (delays) {Delays};
+   \node[control, right=of eqx] (eq) {Eq coefficients};
+   \draw[->] (receive)
+     -| node[lbl, very near start, auto, swap] {V}
+        node[lbl, near end, auto, swap] {i10} (unpackx);
+   \draw[->] (receive)
+     -| node[lbl, very near start, auto] {H}
+        node[lbl, near end, auto] {i10} (unpacky);
+   \draw[->, dotted] (delays) to[lbl, auto, edge label'=i32] (cdelayx);
+   \draw[->, dotted] (delays) to[lbl, auto, edge label=f32] (eqx);
+   \draw[->, dotted] (delays) to[lbl, auto, edge label=i32] (cdelayy);
+   \draw[->, dotted] (delays) to[lbl, auto, edge label'=f32] (eqy);
+   \draw[->, dotted] (eq) to[lbl, auto, edge label'=cf32] (eqx);
+   \draw[->, dotted] (eq) to[lbl, auto, edge label=cf32] (eqy);
+   \draw[->] (quantx) |- node[lbl, auto, swap, near start] {ci8} (pack);
+   \draw[->] (quanty) |- node[lbl, auto, near start] {ci8} (pack);
+
+The mixer frequency is quantised to a 0.32 unsigned fixed-point representation
```

Comment:
I am a bit rusty on this. Does this mean that you use all 32 bits for the fraction part and none for integers? i.e. so the range is [0, 1) or as close to 1 as you can get with 32 bits.

Reply:
Yes.